thriftrw/spec/struct.py

```diff
@@ -409,6 +425,17 @@ def __init__(self, *args, **kwargs):
     return __init__
 
 
+def struct_init_field(self, name, value):
+    field = [f for f in self.type_spec.fields if f.name == name][0]
```

Comment:
This is expensive. You'll iterate through the list of fields defined for the struct for each field being set.